remoteworkitem/tracker.go

```diff
@@ -5,7 +5,7 @@ import "github.com/almighty/almighty-core/models"
 // Tracker represents tracker configuration
 type Tracker struct {
 	models.Lifecycle
-	ID uint64 `gorm:"primary_key"`
+	ID string `gorm:"primary_key"`
```

Comment:
@Ritsyy Those are two different things. app.Tracker != remoteworkitem.Tracker. app.Tracker is the MediaType and remoteworkitem.Tracker is the database model. The TrackerRepository use app.Tracker as input and output. app.Tracker has a string id, which means TrackerRepository.Load etc should take a string id as well. What remoteworkitem.Tracker has is irelevant.